app/controllers/users_controller.rb

```diff
@@ -1,3 +1,31 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user.present?
```

Comment:
Chỗ này mình biết find_by nếu k có nó sẽ trả về @user là nil rồi nên chỉ cần check nil là đc.
```suggestion
    return if @user
```
Cũng k cần dùng tới present?